wiki/Play_style/Alternating/id.md

```diff
@@ -0,0 +1,27 @@
+---
+stub: true
+tags:
+  - keyboard
+  - tap
+  - alternate
+  - alt
+  - play style
+---
+
+# Alternating
+
+*Berbeda dengan [Double tapping](/wiki/Play_style/Double_tapping).*
+
+Alternating adalah gaya ketukan di [osu!standard](/wiki/Game_mode/osu!) dimana pemain menekan circle, menggeser slider, dan memutar spinner dengan Alternating antara dua set tombol di keyboard mereka (`Z` and `X` secara default) satu sama lain [hit objek](/wiki/Hit_object) kurang lebih. Oleh karena itu. Gaya ini paling umum di antara pemain yang lebih berpengalaman dan merupakan salah satu yang paling sulit untuk diadaptasi.
```

Comment:
```suggestion
Alternating adalah sebuah gaya tapping di [osu!](/wiki/Game_mode/osu!) dimana pemain menekan circle, menggeser slider, dan memutar spinner dengan menekan secara bergantian antara dua tombol pada keyboard mereka (`Z` and `X` secara default) satu sama lain [hit object](/wiki/Hit_object) kurang lebih. Oleh karena itu. Gaya ini paling umum di antara pemain yang lebih berpengalaman dan merupakan salah satu yang paling sulit untuk diadaptasi.
```

Reply:
I think that we should keep ```tapping``` in its original place, instead of using something like "ketukan" which might sounds confusing.

Should change ```Alternating``` on this one to ```menekan secara bergantian``` to explain of what it actually is, but not sure if it would be better to keep all the other ```Alternating``` word or change it aswell...

Reply:
i agree with `tapping` but, i still doubt with `menekan secara bergantian`

Reply:
do we have an update on this?

Reply:
ahhhh, i forgot smh